debezium-core/src/main/java/io/debezium/pipeline/meters/StreamingMeter.java

```diff
@@ -86,6 +86,10 @@ public void onEvent(DataCollectionId source, OffsetContext offset, Object key, S
         }
     }
 
+    public void resetLagBehindSource() {
```

Comment:
IMHO the method needs to be declared at `io.debezium.pipeline.metrics.ChangeEventSourceMetricsMXBean` to be visible in JMX or another one and implemented here or elswhere.

Reply:
Right, that is probably better place. I wanted to place it sid-by-side with the global reset but this is better solution.